angular/projects/admin/src/app/admin/form/textbox-rich/textbox-rich.component.ts

```diff
@@ -131,11 +133,42 @@ export class TextboxRichComponent implements MatFormFieldControl<string>, Contro
   }
 
   setDisabledState(isDisabled: boolean): void {
-    this.formEditor.isReadOnly = isDisabled;
+    // this.formEditor.isReadOnly = isDisabled;
```

Comment:
Delete or uncomment

Reply:
Resolved on this commit https://github.com/oddbit/tanam/pull/266/commits/80992f5e27f46e340199dacbbba71d676674b93a